sql/dd/impl/properties_impl.h

```diff
@@ -83,8 +83,14 @@ class Properties_impl : public Properties {
  public:
   Properties_impl() = default;
 
+  Properties_impl(const Properties_impl &o)
```

Comment:
= default;
?

Reply:
"= default" was not sufficient to silence the "error: definition of implicit copy constructor for 'Properties' is deprecated because it has a user-declared destructor" "Rule of 3" error on all platforms.  Only adding an explicit copy constructor solved this.